pages/a-la-francaise.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="fr">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Ohmyfood</title>
+    <meta
+      name="description"
+      content="Ohmyfood, reservez le menu qui vous convient."
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Raleway:wght@300;400;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./assets/css/normalize.css" />
+    <link rel="stylesheet" href="./assets/css/style.css" />
+  </head>
+  <body>
+    <header>
+      <img
+        src="../assets/images/logo/ohmyfood.png"
+        alt="Logo du site Ohmyfood."
+      />
+    </header>
+    <main>
+      <img
+        src="../assets/images/restaurants/toa-heftiba-DQKerTsQwi0-unsplash.jpg"
+        alt="restaurant"
+      />
+      <section id="french-style">
+        <div>
+          <h2>À la française</h2>
+          <i class="fa-regular fa-heart"></i>
+        </div>
+        <h3>ENTRÉES</h3>
+        <ul>
+          <li>
+            <h4>Tartare de poulpe acidulé</h4>
+            <p>Aux zests d'orange</p>
+            <div>25€</div>
+          </li>
+          <li>
+            <h4>Velouté de légumes d'antan</h4>
+            <p>Carotte, panais, topinambourg</p>
+            <div>35€</div>
+          </li>
+          <li>
+            <h4>Soupe à l'oignon</h4>
+            <p>Revisitée</p>
+            <div>20€</div>
+          </li>
+        </ul>
```

Comment:
not a list, use articles with on div for tartare and aux zest, a span for price  and a div for the check